pkg/daemon/daemon.go

```diff
@@ -484,10 +484,14 @@ func (dn *Daemon) nodeStateSyncHandler() error {
 	if reqDrain || !utils.ObjectHasAnnotation(dn.desiredNodeState,
 		consts.NodeStateDrainAnnotationCurrent,
 		consts.DrainIdle) {
-		if err := dn.handleDrain(reqReboot); err != nil {
+		shouldReconcile, err := dn.handleDrain(reqReboot)
```

Comment:
can you rename the variable to something like `inProgress` or `drainInProgress`?  For me `shouldReconcile` looks more like "should continue the current reconcile cycle", but not like "retry on next reconcile cycle" (the meaning which we want, If I understood correctly)

Reply:
sure! done